### PR TITLE
Bug 129 - Make "submetas" work with all sections, not just `p1`

### PR DESCRIPTION
**Pull request by AllegroFox**
I've disabled the no-lonely-if eslint rule on line 379 for now - to me it's clearer if each branch has the same structure in this case, but I can always refactor it!

### CODE REVIEW

**Comment by AllegroFox** [comment]
I don't know why the Travis build is failing here by the way...it's a Prettier failure, and everything is passing locally.

**Comment by Mardak** [comment]
Did you `npm i` ? I see the same error:

```
Checking formatting...
src/content/components/FeatureView/FeatureView.js
Code style issues found in the above file(s). Forgot to run Prettier?
```

**Comment by Mardak** [comment]
Ah, looks like the pattern is to `npm run prettier` and it generates this diff:
```diff
diff --git a/src/content/components/FeatureView/FeatureView.js b/src/content/components/FeatureView/FeatureView.js
--- a/src/content/components/FeatureView/FeatureView.js
+++ b/src/content/components/FeatureView/FeatureView.js
@@ -367,3 +367,5 @@ export class FeatureView extends React.PureComponent {
           if (subMetaMatch.length) {
-            subMetaMatch.forEach(m => result.untriagedBySubMeta[m.id].push(bug));
+            subMetaMatch.forEach(m =>
+              result.untriagedBySubMeta[m.id].push(bug)
+            );
           } else {
```

File: src/content/components/FeatureView/FeatureView.js [hunk 1]
```diff
@@ -248,10 +286,13 @@ export class FeatureView extends React.PureComponent {
   sortByRelease(bugs, subMetas) {
     const result = {
       untriaged: [],
+      untriagedBySubMeta: {},
```
Comment (k88hudson) [inline src/content/components/FeatureView/FeatureView.js]: I think actually it's ok if `untriaged` doesn't get split up by sub meta, there's usually not enough bugs to warrant spliting them up

**Pull request merged**